treeherder/push_health/usage.py

```diff
@@ -28,6 +28,11 @@ def get_latest(facet):
             return {NEED_INVESTIGATION: latest['max'], 'time': item['endTimeSeconds']}
 
 
+def jobs_retriggers(push):
```

Comment:
small change in english, I would prefer the function to be named:
jobs_retriggered